initfiles/etc/DIR_NAME/environment.conf.in

```diff
@@ -36,10 +36,10 @@ mpTraceLevel=0
 
 #Specify location of HPCC PKI public/private key files
 # note: if HPCCPassPhrase specified it must be encrypted
-#HPCCPassPhrase=
-#HPCCCertificateFile=/home/hpcc/.ssh/cert.cer
-#HPCCPublicKeyFile=/home/hpcc/.ssh/id_rsa.pub
-#HPCCPrivateKeyFile=/home/hpcc/.ssh/id_rsa
+#HPCCPassPhrase=XC+0Byt45EYD6qV5++8HW+lgsYB/nGud8FtgrubKdLk=
+#HPCCCertificateFile=/home/hpcc/certificate/certificate.pem
```

Comment:
Looks good, but we might want to do something like this so it's not static and will change if we modify options at build time.

#HPCCCertificateFile=${HOME_DIR}/${RUNTIME_USER}/certificate/certificate.pem

And then similar with the other two files.